File: Me-supraja (1)/Me-supraja/CrEST/Microsoft.Crest.Web.API/CrEST.Models/ServiceMetadata.cs
using CrEST.Data.Models;
using System.Collections.Generic;

namespace CrEST.Models
{
    public class ServiceMetadata
    {
        public List<Supplier> Suppliers { get; set; }
        public List<int> ContractIds { get; set; }
        public List<CrestLevel1> CrestLevel1s { get; set; }
        public List<CrestLevel2> CrestLevel2s { get; set; }
        public List<CrestLevel3> CrestLevel3s { get; set; }
        public List<Itorg> ItOrg { get; set; }
    }
}

File: CrEST/Microsoft.CrEST.UI/Microsoft.CrEST.UI/angularCrest/app/sla/shared/models/serviceMetaData.ts
export class ServiceMetaData {
           public suppliers: Array<Supplier>;
            public contractIds: Array<number>;
            public crestLevel1s: Array<CrestLevel1Metadata>;
            public crestLevel2: Array<CrestLevel2Metadata>;
            public crestLevel3: Array<CrestLevel3Metadata>;
            public itOrg: Array<ITOrg>;
};


export class CrestLevel1Metadata {
            public id: number;
            public service: string;
        
        }


        export class ITOrg {
            public itOrgId: number;
            public itOrgName: string;

        }
        export class CrestLevel2Metadata {
            public id: number;
            public service: string;
           
        }

        export class CrestLevel3Metadata {
            public id: number;
            public Service: string;
          

        }
        export class Supplier {
            public supplierId: number;
            public supplierName: string;
        }
